stable/hazelcast-jet/Chart.yaml

```diff
@@ -1,8 +1,7 @@
-apiVersion: v1
```

Comment:
why was this line removed?

Reply:
I thought that is not required but I've just double-checked this after your reminder, will add this again.